messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/ha/ScaleDownAttributes.java

```diff
@@ -112,6 +122,10 @@ static ScaleDownConfiguration addScaleDownConfiguration(OperationContext context
             List<String> connectorNames = new ArrayList<>(connectors.keys());
             scaleDownConfiguration.setConnectors(connectorNames);
         }
+        ModelNode commitInterval = SCALE_DOWN_COMMIT_INTERVAL.resolveModelAttribute(context, model);
+        if (commitInterval.isDefined()) {
+            scaleDownConfiguration.setCommitInterval(commitInterval.asInt());
```

Comment:
I would handle here the 'IllegalArgumentException' that can be thrown by asInt() with a default

Reply:
It should have been validated but the attribute validation

Reply:
I see, thanks @ehsavoie !